### PR TITLE
[dem] Bugfixes with respect to DEM clusters

### PR DESCRIPTION
**Pull request by maceligueta**
Several errors that were not detected in the past. Now clusters can be run from the new GUI.

### CODE REVIEW

File: applications/DEMApplication/CMakeLists.txt [hunk 1]
```diff
@@ -133,6 +133,7 @@ endif(${INSTALL_TESTING_FILES} MATCHES ON)
 if(${INSTALL_PYTHON_FILES} MATCHES ON)
   get_filename_component (CURRENT_DIR_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME)
   install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/python_scripts DESTINATION applications/${CURRENT_DIR_NAME}  FILES_MATCHING PATTERN "*.py"  PATTERN ".svn" EXCLUDE)
+  install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements DESTINATION applications/${CURRENT_DIR_NAME} FILES_MATCHING PATTERN "*.clu" PATTERN ".svn" EXCLUDE)
```
Comment (philbucher) [inline applications/DEMApplication/CMakeLists.txt:136]:
```suggestion
  install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements DESTINATION applications/${CURRENT_DIR_NAME} FILES_MATCHING PATTERN "*.clu" EXCLUDE)
```
don't think you still need the svn :)

File: applications/DEMApplication/custom_elements/cluster3D.cpp [hunk 2]
```diff
@@ -60,6 +60,9 @@ namespace Kratos {
         RigidBodyElement3D::Initialize(r_process_info);
 
         const double cl = GetGeometry()[0].FastGetSolutionStepValue(CHARACTERISTIC_LENGTH);
+	if(!GetProperties().Has(CLUSTER_INFORMATION)) {
```
Comment (philbucher) [inline applications/DEMApplication/custom_elements/cluster3D.cpp]:
```suggestion
	KRATOS_ERROR_IF_NOT(GetProperties().Has(CLUSTER_INFORMATION)) {
```

File: applications/DEMApplication/CMakeLists.txt (hunk 1 above)
Comment (maceligueta) [reply applications/DEMApplication/CMakeLists.txt:136]: @philbucher : the suggested change would exclude the files I want to copy, as far as I understand. Our goal is to copy the *.clu files to the install directory. Maybe I could forget about the .svn, but then I should remove the EXCLUDE keyword as well...

File: applications/DEMApplication/custom_elements/cluster3D.cpp (hunk 2 above)
Comment (maceligueta) [reply applications/DEMApplication/custom_elements/cluster3D.cpp]: Thanks @philbucher , I will change this, but removing the curly brackets...

File: applications/DEMApplication/CMakeLists.txt (hunk 1 above)
Comment (philbucher) [reply applications/DEMApplication/CMakeLists.txt:136]: ok, I am not the expert here :)

**Review by joaquinirazabal — APPROVED** [review]
Nice work @maceligueta, this solves some important bugs. Look good to me.

**Pull request merged**